ansible/roles/ocp4-workload-rhtr-iot-managed/defaults/main.yaml

```diff
@@ -0,0 +1,34 @@
+# role
```

Comment:
All var names for roles should be prefixed with the role name.

Also, is this only ever going to be used for RHTR? If not, can we make the role name more generic and drop the rhtr?